main/go.mod

```diff
@@ -0,0 +1,7 @@
+module launch/main
+
+go 1.19
+
+replace packages/sorting => ../sorting
```

Comment:
This does not work on every evn where you checkout your code. For example based on some env variables you could get: `package packages/sorting is not in GOROOT`. See following comment on how to structure it.